nucleus/annotation.py

```diff
@@ -575,20 +577,22 @@ def __post_init__(self):
     def from_json(cls, payload: dict):
         return cls(
             labels=payload[LABELS_KEY],
-            taxonomy_name=payload[TAXONOMY_NAME_KEY],
+            taxonomy_name=payload.get(TAXONOMY_NAME_KEY, None),
```

Comment:
If the default taxonomy name is "default" why can't users see this in their API objects?

Reply:
My view is that it's more elegant if we abstract away the whole concept of taxonomies in the case where users just use a default taxonomy but we could also expose the fact that this is using a default taxonomy under the hood. @rkaplan do you have an opinion on whether the default taxonomy should be exposed or hidden?

Reply:
Agree I think the default should ideally be hidden.

Imo, taxonomy names should feel like a slightly more advanced feature above the base case, which we expect to be more common, which is simple classification tasks. For someone walking through an MNIST upload tutorial, to me it feels like they should never need to understand taxonomy names. It's just a category label. Wdyt @ardila ?